packages/smooth_app/lib/generic_lib/buttons/smooth_large_button_with_icon.dart

```diff
@@ -25,7 +23,7 @@ class SmoothLargeButtonWithIcon extends StatelessWidget {
         children: <Widget>[
           Icon(
             icon,
-            color: isDarkMode
+            color: MediaQuery.platformBrightnessOf(context) == Brightness.dark
```

Comment:
It would be better to create a single utility method or an extension for this kind of check